src/driver/postgres/PostgresConnectionOptions.ts

```diff
@@ -76,7 +76,9 @@ export interface PostgresConnectionOptions
     readonly poolErrorHandler?: (err: any) => any
 
     /**
-     * Include notification messages from Postgres server in client logs
+     * Include notification messages from Postgres server in client logs.
+     * Note: This only works with the JavaScript `pg` driver.
+     * The native `pg-native` driver does not emit notice/notification events.
```

Comment:
This does not work with the native driver, but TypeORM prefers the native driver, so in CI it uses the native one, on my machine the JS one, so I didn't find this before.